pkg/devicetwin/dtmanager/membership_test.go

```diff
@@ -0,0 +1,352 @@
+/*
+Copyright 2018 The KubeEdge Authors.
```

Comment:
nit: 2018-> 2019. Also please don't delete blank lines. 
Refer to license in this file https://github.com/kubeedge/kubeedge/blob/master/pkg/edgehub/controller_test.go

Reply:
Modified according to 
https://github.com/kubeedge/kubeedge/blob/master/pkg/edgehub/controller_test.go